h2o-py/h2o/grid/grid_search.py

```diff
@@ -421,6 +424,14 @@ def _handle_build_finish(self, grid, rest_ver=None):
         grid_json = h2o.api("GET /99/Grids/%s" % (grid.dest_key))
         failure_messages_stacks = ""
         error_index = 0
+        # hyper_params = grid_json['hyper_names']
```

Comment:
I assume this is not needed anymore (just an initial attempt)

Reply:
Good catch Michalk.  Fixed!